sql/src/test/java/org/apache/druid/sql/calcite/util/CalciteTests.java

```diff
@@ -377,6 +377,15 @@ public AuthenticationResult createEscalatedAuthenticationResult()
   );
 
   public static final List<InputRow> ROWS1_WITH_NUMERIC_DIMS = ImmutableList.of(
+      createRow(
```

Comment:
Hmm, it seems like this change is causing some unrelated test failures

Reply:
Seems like the VarianceSqlAggregatorTest is using this data too and when the ordering of the rows changed, the variance also changed. I created a new datasource that have the numeric dim first for my test and changed the numfoo datasource back to how it was. The reason I wanted to have numeric null first is because the ANY will select the first row and skip everything after. So if the first row is not null, then there is not really any point in testing (if we want to test the numeric null stuff)

Reply:
Actually, I think it's fine to just test with the same numfoo datasource (with first row being non-null)